### ISSUE TITLE
How to best handle static (e.g. _locales) files

### ISSUE THREAD

**Issue by berstend** [kevincharm/parcel-plugin-web-extension#8]
Great project! 👍 

I was wondering how to best handle static files that are not referenced anywhere.

E.g. the `_locales` folder. Currently I use a package.json subtask to specifically copy over some files to the build directory.

Parcel doesn't seem to support that currently: https://github.com/parcel-bundler/parcel/issues/1411

What's your take on this, how would you handle e.g. `_locales` files currently? :-)

Best

**Comment by berstend** [comment]
Agreed, seems like `_locales` is the only case where such specific workaround is required - would be great to support this as it's quite commonly used.

Unfortunately I have to finish a project right now, otherwise I'd give the implementation a shot :-)

For now my workaround using npm scripts works sufficiently (using `cpx`):

```json
{
  "scripts": {
    "build": "export NODE_ENV=development && yarn run build:clean && yarn run locales && parcel build --no-minify src/manifest.json -d build/${NODE_ENV}/",
    "build:production": "export NODE_ENV=production && yarn run build:clean && yarn run locales:copy && parcel build --no-source-maps --no-cache src/manifest.json -d build/${NODE_ENV}/",
    "build:clean": "rimraf build/${NODE_ENV}/",
    "watch": "export NODE_ENV=development && yarn run build:clean && yarn run locales:copy && parcel watch src/manifest.json -d build/${NODE_ENV}/",
    "locales:copy": "cpx 'src/**/*.json' build/${NODE_ENV}/",
  }
}
```

I really like the project so I'll definitely plan to contribute in the future. :-)

**Comment by kevincharm** [comment]
Thanks for sharing that! That looks simple enough. Will work on something tomorrow 😃

**Comment by kevincharm** [comment]
Hey @berstend please check out the PR when convenient and let me know how that works for you.

**Comment by berstend** [comment]
Oh wow, that was quick - thanks a lot @kevincharm 👍

I've reviewed and (successfully) tested your PR. :-)

**Comment by kevincharm** [comment]
No worries, thanks a lot for testing it @berstend :)
Released with v1.3.1

**Comment by loretoparisi** [comment]
In my case I did in this way

```json
{
"scripts": {
    "watch": "cross-env NODE_ENV=development && yarn run locales:copy && parcel index.html --no-hmr --open",
    "build": "cross-env NODE_ENV=production && yarn run locales:copy && parcel build index.html --no-minify --public-url ./",
    "locales:copy": "cp examples/*.md dist/"
 }
}
```
